data/json/items/comestibles/drink.json

```diff
@@ -1042,6 +1042,30 @@
     "relative": { "fun": 1 },
     "use_action": [  ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_green",
+    "name": "green tea",
+    "copy-from": "tea",
+    "color": "green",
+    "fatigue_mod": 3,
```

Comment:
Green tea is a stronger stimulant than black.

Reply:
Though both contain caffeine, black tea usually has more — making green the better choice for people sensitive to this stimulant. Furthermore, green tea contains more L-theanine, an amino acid that’s calming and can balance the effects of caffeine. 
Source: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4787341/

Reply:
I stand corrected, apparently the teas I used in undergrad were not representative. Nevertheless, the caffeine levels aren't anywhere near as disparate as you have here. I'd suggest at most dropping it to 5 if black tea is 6.

Reply:
I can live with that. Will change it accordingly.